### PR TITLE
add chkupdate.py to more packages

### PR DESCRIPTION
**Pull request by getchoo**
adds a chkupdate.py script to 12 out of the 33 (applicable by my count :p) packages listed in #26

maybe i'll work on some others later, these were just the easiest to add tonight

### CODE REVIEW

**Review by lleyton — DISMISSED** [review]

File: anda/apps/openasar-canary/chkupdate.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+f = open(SPEC, "r")
+
+matches = re.findall(REGEX_SHA, txt := f.read())
+if not len(matches):
+    exit(f"{NAME}: Failed to match regex!")
+cur = matches[0]
+
+if sha == cur:
+    exit(f"{NAME}: Up to date!")
+print(f"{NAME}: {cur} -> {sha}")
```
Comment (windowsboy111) [inline anda/apps/openasar-canary/chkupdate.py:23]: so this file writes nothing?

**Review by windowsboy111 — CHANGES_REQUESTED** [review]
overall the requests.get() thingy looks ugly but meh

File: anda/apps/openasar-canary/chkupdate.py (hunk 1 above)
Comment (getchoo) [reply anda/apps/openasar-canary/chkupdate.py:23]: yep, the version is already determined with [%{autogitdate}](https://github.com/terrapkg/packages/blob/44cf5b5a477d3100daee78cca28c344a685fe750/anda/apps/openasar-canary/openasar-canary.spec#L4) and the [source](https://github.com/terrapkg/packages/blob/44cf5b5a477d3100daee78cca28c344a685fe750/anda/apps/openasar-canary/openasar-canary.spec#L10) just points to the continuously updated nightly release upstream. the goal of this script was to just trigger a rebuild on each new commit (which should signify a new nightly release), which is why i started defining the commit in the spec file

**Comment by getchoo** [comment]
> overall the requests.get() thingy looks ugly but meh

that's just how black formats it. i could move the string slices to a different line but it would be kinda pointless

File: anda/apps/openasar-canary/chkupdate.py (hunk 1 above)
Comment (windowsboy111) [reply anda/apps/openasar-canary/chkupdate.py:23]: I don't think builds will trigger with no files changed in the dir…

Comment (getchoo) [reply anda/apps/openasar-canary/chkupdate.py:23]: 🤦🏻 i see what you mean now. i'll push another commit now

...maybe i shouldn't be doing this at 4am

**Review by windowsboy111 — APPROVED** [review]

**Review by lleyton — APPROVED** [review]

**Pull request merged**